apidoc/Titanium/Media/Media.yml

```diff
@@ -1669,6 +1669,10 @@ properties:
     type: Number
     constants: Titanium.UI.iPad.POPOVER_ARROW_DIRECTION*
     platforms: [ipad]
+  - name: allowMultiple
+    summary: Specifies if the user should be able to select multiple photos.
+    type: Boolean
+    platforms: [android]
```

Comment:
Missing `since` version. Since we are pretty settled for the 5.4.0 feature freeze, it likely goes into 6.0.0. @ashcoding decides!